base_tier_validation/readme/HISTORY.rst

```diff
@@ -0,0 +1,78 @@
+12.0.3.3.1 (2019-12-02)
```

Comment:
Great, I didn't think you will pull the whole history before what you did though :)